core/ui/src/main/kotlin/com/google/samples/apps/nowinandroid/core/ui/NewsResourceCard.kt

```diff
@@ -240,11 +237,9 @@ fun NotificationDot(
 }
 
 @Composable
-fun dateFormatted(publishDate: Instant): String = DateTimeFormatter
-    .ofLocalizedDate(FormatStyle.MEDIUM)
-    .withLocale(Locale.getDefault())
-    .withZone(LocalTimeZone.current.toJavaZoneId())
-    .format(publishDate.toJavaInstant())
+fun dateFormatted(publishDate: Instant): String = DateFormat
+    .getDateInstance(DateFormat.MEDIUM, Locale.getDefault())
+    .format(publishDate.toEpochMilliseconds())
```

Comment:
`DateFormat` is not thread-safe or immutable and `DateTimeFormatter` is. 

ideally, the formatter would be `remember`'d (and thread-safe and immutable)

Reply:
Thank you to review it.
Let me says my opinion.

I checked the java [documentation](https://docs.oracle.com/javase/8/docs/api/java/text/DateFormat.html), it said
> Synchronization
>
> Date formats are not synchronized. It is recommended to create separate format instances for each thread. If multiple threads access a format concurrently, it must be synchronized externally.

I don't think my DateFormat is called on multiple thread, it's only called on main thread.
And It has been created new DateFormat instance as `NewsResourceCardExpanded` is composed.